### PR TITLE
Fix Checkstyle Issues

### PR DESCRIPTION
**Pull request by FieryPhoenix909**
Fixes 15+ Checkstyle issues.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Can one of the admins please verify this patch?

File: engine/src/main/java/org/terasology/world/internal/WorldInfo.java [hunk 1]
```diff
@@ -82,8 +82,7 @@ public void setWorldGenerator(SimpleUri worldGenerator) {
     }
 
     @Override
-    public String toString()
-    {
+    public String toString(){
```
Comment (llvieira) [inline engine/src/main/java/org/terasology/world/internal/WorldInfo.java]: You also have to add a blank space between `()` and `{` as you did in other methods.

File: engine/src/main/java/org/terasology/world/sun/DefaultCelestialSystem.java [hunk 2]
```diff
@@ -72,7 +72,8 @@ public float getSunPosAngle() {
     }
 
     @Override
-    public boolean isSunHalted () { return haltSunPosition; }
+    public boolean isSunHalted() {
+        return haltSunPosition; }
```
Comment (llvieira) [inline engine/src/main/java/org/terasology/world/sun/DefaultCelestialSystem.java]: The last method bracket should be one line below.

**Comment by FieryPhoenix909** [comment]
The edits have been made!

File: engine/src/main/java/org/terasology/logic/console/ConsoleImpl.java [hunk 3]
```diff
@@ -124,8 +124,7 @@ private void addErrorMessage(String message) {
      * @param newLine    A boolean: True causes a newline character to be appended at the end of the message. False doesn't.
      */
     @Override
-    public void addMessage(String message, boolean newLine)
-    {
+    public void addMessage(String message, boolean newLine){
```
Comment (llvieira) [inline engine/src/main/java/org/terasology/logic/console/ConsoleImpl.java]: Oh I forgot this one. So please, add a space here too.

**Pull request merged**